.github/workflows/windows.yml

```diff
@@ -41,11 +41,11 @@ jobs:
           flex 
           dos2unix 
           mingw-w64-ucrt-x86_64-gperf
-    - shell: msys2 {0}
+          
+    - name: Build with cmake
       run: |
-        set MSYSTEM=UCRT64
-        cmake.exe -B ${{github.workspace}}/build -G 'Unix Makefiles' -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}}
-        cmake.exe --build ${{github.workspace}}/build -j`nproc` --config ${{env.BUILD_TYPE}}
+        cmake -B ${{github.workspace}}/build -G 'Unix Makefiles' -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}}
```

Comment:
Try using `-G "MinGW Makefiles"` option The Unix Makefiles generator may not work within MSYS2.